civihr_default_theme/scss/pages/_hr-details.scss

```diff
@@ -65,3 +65,198 @@
     }
   }
 }
+
+.view-mydetails-personal {
+  .contact-id span {
+    @include views-label;
+  }
+
+  .pane-block {
+    border-bottom: 1px solid $gray-light;
+    margin-bottom:0px;
+  }
+
+  .views-row {
+    min-height: 80px;
+    padding-left: 100px;
+    position: relative;
+  }
+
+  .views-field-image-URL {
+    left: 0px;
+    position: absolute;
+    top: 0px;
+  }
+
+  .no-user-image {
+    background: url(/sites/all/modules/civihr-custom/civihr_employee_portal/images/profile-default.png) center no-repeat;
+    background-size: 80px 80px;
+    overflow: hidden;
+    text-indent: -1000px;
+  }
+
+  .views-field-image-URL img,
+  .no-user-image {
```

Comment:
there's something odd going on here. The profile picture is already styled and has a very specific markup
<img width="400" alt="profile-picture-markup" src="https://user-images.githubusercontent.com/6400898/42083387-09953e14-7b8b-11e8-9a93-44b0de1d7cc8.png">

but i have the feeling we're redefining everything from scratch

Reply:
I'm not using the same view because dashboard already needs this view. It is a different view in purpose to avoid touching anything else but My Details page.